src/delfitlm/settings.py

```diff
@@ -24,6 +24,12 @@
 # gets the key from the runtime environment (docker-compose-deploy)
 # change_me is default if a value is not specified
 SECRET_KEY = os.environ.get('SECRET_KEY', 'change_me')
+POSTGRES_DB         = os.environ.get('POSTGRES_DB', 'postgres')
+POSTGRES_USER       = os.environ.get('POSTGRES_USER', 'postgres')
+POSTGRES_PASSWORD   = os.environ.get('POSTGRES_PASSWORD', 'postgres')
+POSTGRES_HOST       = os.environ.get('POSTGRES_HOST', 'localhost')
+POSTGRES_PORT       = int(os.environ.get('POSTGRES_PORT', 5432))
+
```

Comment:
How do you handle the db initialization the first time (with an empty data folder)? Are the tables automatically created or is there a particular step to follow to get the system up and running? Probably the same comment can also address the user management tables (who is the default admin user and password when the database is created?)

Reply:
I was about to try a migration, but this branch has no db tables defined. I think it should work by running the db container then the we do `python src/manage.py makemigrations`  &&  `python src/manage.py migrate`. That should create the tables. The admin can be created with `python src/manage.py createsuperuser` to access the django admin interface.

Reply:
I'll look into creating and applying migrations at the start of the container

Reply:
There a table for Delfin3xt telemetry now on the main branch, you could play around with that one.

Reply:
I have ran the migrations from the main branch and I can see it back in the logs:

```
hidde@DESKTOP-D43AJV9 /m/c/d/DelfiTLM (postgres-db) [1]>
docker logs delfitlm-app-1
Operations to perform:
  Apply all migrations: admin, auth, contenttypes, delfin3xt, sessions
Running migrations:
  Applying contenttypes.0001_initial... OK
  Applying auth.0001_initial... OK
  Applying admin.0001_initial... OK
  Applying admin.0002_logentry_remove_auto_add... OK
  Applying admin.0003_logentry_add_action_flag_choices... OK
  Applying contenttypes.0002_remove_content_type_name... OK
  Applying auth.0002_alter_permission_name_max_length... OK
  Applying auth.0003_alter_user_email_max_length... OK
  Applying auth.0004_alter_user_username_opts... OK
  Applying auth.0005_alter_user_last_login_null... OK
  Applying auth.0006_require_contenttypes_0002... OK
  Applying auth.0007_alter_validators_add_error_messages... OK
  Applying auth.0008_alter_user_username_max_length... OK
  Applying auth.0009_alter_user_last_name_max_length... OK
  Applying auth.0010_alter_group_name_max_length... OK
  Applying auth.0011_update_proxy_permissions... OK
  Applying auth.0012_alter_user_first_name_max_length... OK
  Applying delfin3xt.0001_initial... OK
  Applying sessions.0001_initial... OK
Watching for file changes with StatReloader
```

And also in pgAdmin:
![afbeelding](https://user-images.githubusercontent.com/15870306/144131253-fc40d89b-5982-4e79-b034-7c6f670e6e94.png)

It's looking very good to me!